tool/tbot/main.go

```diff
@@ -158,6 +166,18 @@ func Run(args []string, stdout io.Writer) error {
 		"Arguments to `tsh proxy ...`; prefix with `-- ` to ensure flags are passed correctly.",
 	))
 
+	sshProxyCmd := app.Command("ssh-proxy-command", "An OpenSSH/PuTTY proxy command").Hidden()
+	sshProxyCmd.Flag("destination-dir", "The destination directory with which to authenticate tsh").StringVar(&cf.DestinationDir)
+	sshProxyCmd.Flag("cluster", "The cluster name. Extracted from the certificate if unset.").StringVar(&cf.Cluster)
+	sshProxyCmd.Flag("user", "The remote user name for the connection").Required().StringVar(&cf.User)
+	sshProxyCmd.Flag("host", "The remote host to connect to").Required().StringVar(&cf.Host)
+	sshProxyCmd.Flag("post", "The remote port to connect on.").Required().StringVar(&cf.Port)
+	sshProxyCmd.Flag("proxy-server", "The Teleport proxy server to use, in host:port form.").Required().StringVar(&cf.ProxyServer)
+	sshProxyCmd.Flag("tls-routing", "Whether the Teleport cluster has tls routing enabled.").Required().BoolVar(&cf.TLSRoutingEnabled)
+	sshProxyCmd.Flag("connection-upgrade", "Whether the Teleport cluster requires an ALPN connection upgrade.").Required().BoolVar(&cf.ConnectionUpgradeRequired)
+	sshProxyCmd.Flag("proxy-template-file", "The path to a file containing proxy templates to be evaluated.").StringVar(&cf.TSHConfigPath)
```

Comment:
Nit: slight preference for `--proxy-templates` or `--proxy-template` but the most minor preference ever.